src/map_strings.hpp

```diff
@@ -28,18 +28,33 @@
 
 // seqan3 headers
 #include <seqan3/core/debug_stream.hpp>
-#include <seqan3/alphabet/all.hpp>
-#include <seqan3/range/view/all.hpp>
-#include <seqan3/std/ranges>
-#include <seqan3/io/sequence_file/all.hpp>
-#include <seqan3/search/fm_index/fm_index.hpp>
-#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/search/fm_index/all.hpp>
 #include <cereal/archives/binary.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/range/views/all.hpp>
+#include <seqan3/search/search.hpp>
+//#include <seqan3/std/filesystem>
+#include <seqan3/std/ranges>
+
+// Bifrost headers
+#include <bifrost/ColoredCDBG.hpp>
 
 namespace py = pybind11;
-using namespace seqan3;
+//using namespace seqan3;
 
-typedef fm_index<true, default_sdsl_index_type> fasta_fm_index;
+// fmindex typedef
+using cust_sdsl_wt_index_type = sdsl::csa_wt<sdsl::wt_blcd<sdsl::bit_vector,
+        sdsl::rank_support_v<>,
+        sdsl::select_support_scan<>,
+        sdsl::select_support_scan<0>>,
+        16,
+        10000000,
+        sdsl::sa_order_sa_sampling<>,
+        sdsl::isa_sampling<>,
+        sdsl::plain_byte_alphabet>;
+typedef seqan3::fm_index<seqan3::dna5, seqan3::text_layout::collection, cust_sdsl_wt_index_type> fasta_fm_index;
+using seqan3::operator""_dna5;
 
 // Constants
 const std::string VERSION = "1.1.0";
```

Comment:
We should remove this. Can you check where it's used (if it is used) and replace with `VERSION_INFO `